source/manuals/accessibility.html.md.erb

```diff
@@ -0,0 +1,115 @@
+
+Non sighted users can also skim read a page. Screen readers can announce content by element type, such as headings, paragraphs or links. For example, if a screen reader user expects a page to contain data in a table element, such as a train timetable, they might start by reading through all the tables on a page.
+
+This is why semantic markup and good heading structure are important when building accessible services.
+
+### Automated testing
+
+Accessibility testing tools such as [aXe](https://www.deque.com/axe/) and [Lighthouse](https://developers.google.com/web/tools/lighthouse/) are useful for finding issues, but [automated testing can only find around 30% of likely accessibility problems](https://alphagov.github.io/accessibility-tool-audit/). Automated testing shouldn’t be the only part of accessibility testing a service.
+
+### Browser compatibility testing
+
+Services should be thoroughly tested in all browsers they are required to be compatible with, including mobile devices and accessibility tools such as screen readers. Emulated browser testing services are available, but some testing on real mobile devices is also recommended.
+
+For reference, these are the [browsers that GOV.UK is required to be compatible with](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices).
+
+### Test content served by third party systems
+
+If a service is built using an existing or third party system, such as a content management system or JavaScript framework, it should be tested for accessibility. Accessibility compliance cannot be guaranteed by any system, as even small changes can introduce accessibility barriers.
+
+### Testing with real users
+
+Services should be tested by real users to evaluate how accessible they are. This can be with members of the public as well as specialist organisations who provide accessibility testing services.
+
+Services can also be tested using an [empathy lab](https://gds.blog.gov.uk/2018/06/20/creating-the-uk-governments-accessibility-empathy-lab/), such as the one at GDS, which is designed to provide a simulation of using the web with accessibility issues so developers can better understand what this is like.
+
+Testing with real users can be a time consuming process. It is recommended that this is done only after other accessibility testing has occurred first.
+
+### Consider using the GOV.UK Design System
+
+The [GOV.UK Design System](https://design-system.service.gov.uk/) is a collection of website styles, components and patterns designed to be used to build government services. It provides pre-built website markup that has been developed specifically to be accessible. Using it will also save development time and provide a look and feel consistent with other government services.
+
+While it may not have a full set of components for every service, it provides a solid foundation to work from that can be expanded to suit a services individual needs.
```

Comment:
"While it may not have a full set of components for every service, **but** it provides...

Reply:
I think you'd either write the sentence like this:

`While it may not have a full set of components for every service, it provides a solid foundation..`

or:

`It may not have a full set of components for every service, but it provides a solid foundation..`

The first is two things that are true at the same time, so there's not need for a `but`. The second is a statement followed by a counterpoint. I chose the first because `While` is a softer opening and implies that a contrary is coming (and I like the flow).